conan/tools/apple/xcodedeps.py

```diff
@@ -86,38 +129,27 @@ def generate(self):
         for generator_file, content in generator_files.items():
             save(generator_file, content)
 
-    def _config_filename(self):
-        # Default name
-        props = [("configuration", self.configuration),
-                 ("architecture", self.architecture)]
-        name = "".join("_{}".format(v) for _, v in props if v is not None)
-        return name.lower()
-
     def _vars_xconfig_file(self, dep, name, cpp_info):
         """
-        content for conan_vars_poco_x86_release.xcconfig, containing the variables
+        returns a .xcconfig file with the variables definition for one package for one configuration
         """
-        # returns a .xcconfig file with the variables definition for one package for one configuration
 
-        pkg_placeholder = "$(CONAN_{}_ROOT_FOLDER_{})/".format(name, self.configuration)
         fields = {
             'name': name,
-            'configuration': self.configuration,
-            'architecture': self.architecture,
-            'root_folder': dep.package_folder,
-            'bin_dirs': " ".join('"{}"'.format(os.path.join(dep.package_folder, p)) for p in cpp_info.bindirs),
-            'res_dirs': " ".join('"{}"'.format(os.path.join(dep.package_folder, p)) for p in cpp_info.resdirs),
-            'include_dirs': " ".join('"{}"'.format(os.path.join(dep.package_folder, p)) for p in cpp_info.includedirs),
-            'lib_dirs': " ".join('"{}"'.format(os.path.join(dep.package_folder, p)) for p in cpp_info.libdirs),
+            'bin_dirs': " ".join('"{}"'.format(os.path.join(p)) for p in cpp_info.bindirs),
```

Comment:
removing dep.package_folder, now folders are absolute

Reply:
This would be great to do in 1.X if possible, not just in 2.0

Reply:
Then, we can remove all the `os.path.join`, right?

```suggestion
            'bin_dirs': " ".join('"{}"'.format(p) for p in cpp_info.bindirs),
```